.circleci/config.base.yml

```diff
@@ -525,6 +528,38 @@ jobs:
           path: ~/repo/packages/amplify-migration-tests/amplify-migration-reports
     working_directory: ~/repo
 
+  amplify_migration_tests_v10:
+    environment:
+      AMPLIFY_PATH: /home/circleci/.amplify/bin/amplify
```

Comment:
this is where the amplify cli is installed when using npm -g install, for v10 and up (or anything that uses the packaged cli); thats why its different here